### PR TITLE
feat!: remove fetch (moved to vrs anvil toolkit)

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

**Comment by jsstevenson** [comment]
@quinnwai I assume this is the plan in the short term? We could always break this out differently at some point but it probably makes sense to consolidate it *somewhere* for the time being

File: src/vrsix/fetch.py [hunk 1]
```diff
@@ -1,57 +0,0 @@
...
-    result = conn.cursor().execute(
-        f"SELECT vrs_id, chr, pos FROM vrs_locations WHERE vrs_id IN ({placeholders})",  # noqa: S608
-        vrs_ids,
-    )
-    data = result.fetchall()
-    conn.close()
-    return [(f"ga4gh:VA.{row[0]}", row[1], row[2]) for row in data]
-
-
-def fetch_by_pos_range(
```
Comment (quinnwai) [inline src/vrsix/fetch.py:40]: This is not contained in the toolkit, did you still want to remove it for now?

File: tests/test_fetch.py [hunk 2]
```diff
@@ -1,52 +0,0 @@
-from pathlib import Path
-
-import pytest
-
-from vrsix import fetch
-
-
-@pytest.fixture()
```
Comment (quinnwai) [inline tests/test_fetch.py:8]: this whole file would probably be good to add to a test on the vrs toolkit side, would you be comfortable adding this?

**Review by quinnwai — APPROVED** [review]
Left some comments, so long as it's good with you to deprecate some existing functions temporarily then go for it. Only thing is that I'd want to add some of the fetching tests to VRS Toolkit if that'd be easy for you to do (outside of PR scope prolyl).

File: tests/test_fetch.py (hunk 2 above)
Comment (jsstevenson) [reply tests/test_fetch.py:8]: Yeah, definitely. Sounds good to me.

**Pull request merged**